spec/serializers/topic_view_serializer_spec.rb

```diff
@@ -64,4 +66,44 @@
       end
     end
   end
+
+  let(:user) { Fabricate(:user) }
```

Comment:
Dup of https://github.com/discourse/discourse/pull/5610/files#diff-5790c3234f6398446eb0fc1ebf01245aR10